tracker/telegram/bot.py

```diff
@@ -60,9 +63,11 @@ async def start_message(message: Message) -> None:
     :param message: Message that starts the bot.
     :return: None
     """
+    message_text = TEMPLATES["greeting"].substitute(
```

Comment:
Create dataclass for TemplateNames: with `greeting` `repo_header` and etc use them. If you have everything in one place it's easier to maintaince in the future